gateway/tests/e2e/providers/common.rs

```diff
@@ -715,7 +718,7 @@ pub async fn test_simple_streaming_inference_request_with_provider(provider: E2E
 pub async fn test_inference_params_inference_request_with_provider(provider: E2ETestProvider) {
     let episode_id = Uuid::now_v7();
 
-    let payload = json!({
+    let mut payload = json!({
```

Comment:
no need to make this mutable if we just always pass the credentials which can be empty